plugin/accel_embree/CMakeLists.txt

```diff
@@ -22,6 +22,7 @@ if (embree_FOUND)
         SOURCES
             "embree.h"
             "accel_embree.cpp"
+            "embree.cpp"
```

Comment:
The file name of embree.h might be confusing with official header. you can use something more explicit like embree_params.h